src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -89,7 +103,7 @@ private ReadFilterLibrary(){ /*no instance*/ }
                     (! read.isUnmapped() && read.getContig().equals(read.getMateContig()));}}
 
     /**
-     * Reads that have a mapped mate and both mate and read are on different same strands (ie the usual situation).
+     * Keep reads that have a mapped mate and both mate and read are on different same strands (i.e. the usual situation).
```

Comment:
Changed with a small modiication to match the "Keep" convention.

Regarding your question, I do not know why the coverse situation is not included...